src/extension.ts

```diff
@@ -37,6 +39,8 @@ export async function activate(ctx: vscode.ExtensionContext) {
   const root = vscode.workspace.workspaceFolders[0].uri.fsPath;
   const buildDir = workspaceRelative(extensionConfiguration("buildFolder"));
 
+  workspaceState.update("mesonbuild.buildDir", buildDir);
```

Comment:
Can we call it `mesonbuild.buildFolder`? Rename the variable too.

Reply:
Personally, I do not like the Microsoft newspeak that replaces directory with folder. The `buildDir` name is pervasive in the repository.

```
[18:34 wolf@oberon:~/vscode-meson]% git grep buildDir | wc -l
78
[18:34 wolf@oberon:~/vscode-meson]% git grep buildFolder | wc -l
7
```

For example, the current main branch:
https://github.com/mesonbuild/vscode-meson/blob/68f18fb6664234a06b4496101368e3eaee0921fe/src/utils.ts#L67

Reply:
Ok. Let's work on replacing the setting in the future. No big deal